parent/pom.xml

```diff
@@ -1278,18 +1279,19 @@
             </dependency>
           </dependencies>
 
+<!-- Disabled due to https://issues.apache.org/jira/projects/MDEP/issues/MDEP-774 and https://issues.apache.org/jira/browse/MDEP-753
```

Comment:
:x: Instead of commenting it, let's hard code the `<skip>true</skip>` for executions. That way, when it's update-able, hopefully dependabot will tell us. If we comment it out, I think it won't automatically notify us.